helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -418,16 +418,9 @@ internal fun String.wrapAt(column: Int): String {
 
     var text = this
     while (text.isNotEmpty()) {
-        val spaceIndex = text.indexOf(' ', column)
-
-        var index = spaceIndex
-        while (index > column) {
-            // Find the last space before the column.
-            while (index > 0 && text[--index] != ' ');
-        }
-
-        // If there is no space before the column, take the first space after the column.
-        if (index == 0) index = spaceIndex
+        val firstSpaceAfterColumnIndex = text.indexOf(' ', column)
+        val lastSpaceBeforeColumnIndex = text.substring(0, column).lastIndexOf(' ')
```

Comment:
I believe this could even be simplified further by only using `val lastSpaceBeforeColumnIndex = text.lastIndexOf(' ', column - 1)`.

Reply:
Hm, `column - 1` would be the `startIndex` parameter. So it couldn't match a space before that index?!

Reply:
I see, thanks! Nice simplification btw!